parallel_test_runner/parallel_test_runner.cs

```diff
@@ -21,12 +21,11 @@ enum Granularity {
     return (T)Enum.Parse(typeof(T), value, true);
   }
 
-  const string vsinstr =
+  const string vsinstr_ =
```

Comment:
`const` in C# is more like `static const` or `constexpr` in C++, do we really want a trailing underscore?